File: javascript/firstnseries.js
var readln = (typeof readline === 'function') ? (readline) : (function() {
   importPackage(java.io);
   importPackage(java.lang);
   var stdin = new BufferedReader(new InputStreamReader(System['in']));
   return function() {
     return String(stdin.readLine());   
   };                                    
 }());
System.out.print("How many term in a series do you want to print?");
var Count = readln(Count);
var string = "The first " + Count + " terms in a series are";
var Term = 1;
for(var Multiple = 1; Multiple <= Count; Multiple++)
{
  Term = 1;
  for(var Counter = 1; Counter <= Multiple; Counter++)
  {
    Term = Term * Multiple;
  }
  string = string + " " + Term;
}
string = string + ".";
print(string);

File: java/Firstnseries.java
import java.util.*;
class Firstnseries
{
  public static void main(String[] args)
  {
      System.out.print("How many term in a series do you want to print?");
      Scanner sc = new Scanner(System.in);
      int Count = sc.nextInt();
      System.out.print("The first " + Count + " terms in a series are");
      int Term = 1;
      for(int Multiple = 1; Multiple <= Count; Multiple++)
      {
         Term = 1;
         for(int Counter = 1; Counter <= Multiple; Counter++)
         {
           Term = Term * Multiple;
         }
         System.out.print(" " + Term);
      }
      System.out.print(".\n");
  }
}